packages/sourcecred/src/api/grainConfig.js

```diff
@@ -14,6 +14,8 @@ import {
   parser as bundledGrainIntegrationParser,
 } from "./bundledGrainIntegrations";
 
+// NOTE: This type is deprecated since v0.10.0 and should not be modified
```

Comment:
Didn't I introduce this in 0.10.0 for future use? You should add accountingEnabled to this as well and remove the comment.